src/store/reducer/participants.ts

```diff
@@ -55,6 +60,26 @@ export const participantsReducer = (state: ParticipantsState = null, action: Red
           user: action.user,
         },
         others: [...state!.others],
+        focusInitiator: state!.focusInitiator,
```

Comment:
If I get this right, this is not needed. `...state` returns the whole state and the elements afterwards are the modified ones, so if the EditSelf action has something to do with the focusInitiator, this would be right, but I dont think it does. You are welcome to correct me if i'm wrong :)

Reply:
I'm not sure about the rest of the `focusInitiator` additions some lines above, I think it's worth checking them too